src/compiler/interface/CallSignatureDeclaration.ts

```diff
@@ -23,4 +24,12 @@ export class CallSignatureDeclaration extends CallSignatureDeclarationBase<ts.Ca
     remove() {
         removeInterfaceMember(this);
     }
+
+    /**
+     * Gets the structure equivalent to this node.
+     */
+    getStructure(): CallSignatureDeclarationStructure {
+        return callBaseGetStructure<CallSignatureDeclarationSpecificStructure>(CallSignatureDeclarationBase.prototype, this, {
+        });
+    }
```

Comment:
Needs tests.

Reply:
do you need me to add these missing tests ?

Reply:
@cancerberoSgx sure, that would be very helpful. Thanks! Just copy what's done in [classDeclarationTests.ts](https://github.com/dsherret/ts-simple-ast/blob/162276a6aed164bc1191a04867b8a451e5422b51/src/tests/compiler/class/classDeclarationTests.ts#L1355). Basically, one test for checking the structure when it doesn't have anything and another for when it has everything.

* Don't bother testing the structures of any children that can also get a structure. Just strip that child structure down to the bare minimum (ex. when testing the class just look at the method names in the class). Then write tests separately for the method's structure.
* Only test getting the structure and not filling or anything else. The tests should be focused on what they're testing.

When finished writing tests for a file please commit and push it right away. That way if we're both working in parallel we won't overlap work and if we do it's not that big of a deal because only one file would have changed.

Thanks a lot once again! 😄